rdf-delta-server-local/src/main/java/org/seaborne/delta/server/local/patchstores/zk/PatchLogIndexZk.java

```diff
@@ -38,48 +33,30 @@
 import org.seaborne.delta.server.local.LogEntry;
 import org.seaborne.delta.server.local.PatchStore;
 import org.seaborne.delta.server.local.patchstores.PatchLogIndex;
-import org.seaborne.delta.zk.Zk;
+import org.seaborne.delta.zk.UncheckedZkConnection;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /** State control for a {@link PatchStore} */
 public class PatchLogIndexZk implements PatchLogIndex {
-    private static Logger LOG = LoggerFactory.getLogger(PatchLogIndexZk.class);
+    private static final Logger LOG = LoggerFactory.getLogger(PatchLogIndexZk.class);
 
     private final Object lock = new Object();
-    private final CuratorFramework client;
-
-    private /*final*/ String instance;
+    private final UncheckedZkConnection client;
 
-    private final DataSourceDescription dsd;
     private final String logName;
     private final String statePath;
     private final String lockPath;
     private final String lockStatePath;
     private final String versionsPath;
     private final String headersPath;
-    private final InterProcessLock zkLock;
 
     // null => no watching.
     //private Watcher logStateWatcher = null;
     private final Watcher logStateWatcher;
 
```

Comment:
The following two fields negate each other rendering them useless. They also prevent the deleted block starting at old line 134 below from ever running.